Answers.md

```diff
@@ -1 +1,42 @@
 
+1. Describe the biggest difference between `.forEach` & `.map`.
+
+The .forEach method iterates over each index searching for a specified parameter and returns it as an array.
+The .Map method returns a new array with the index being altered by a function.
+
+2. What is the difference between a function and a method?
+
+The function is the instruction set with the actions within. 
+
+The method is whats used to call on the function. for example, the map method takes in the parameters and is used to call on the map function while passing the parameters to it to be used.
+
+
+3. What is closure?
+
+There are two types of closure global and local. The closure allows for a function to keep its actions isolated from affecting the global scope are other functions unless specifically instructed to do so through call back or calling on a variable located outside its scope.
```

Comment:
You're confusing closure with scope here. A closure is just a function nested inside another function such that the child function has access to the scope of the parent, but the parent does not have access to the scope of the child. Please review these two concepts (closure and scope) in the training kit and make sure that you understand them. If you have any questions, please let me know. Thank you!